packages/frontend/src/components/Game/GameBoard/TileGrid/index.test.jsx

```diff
@@ -9,7 +9,7 @@ describe('TileGrid component', () => {
 
     const { screen } = setupTests(TileGrid, { props: { tiles } });
 
-    const allTiles = screen.getAllByTestId(/tile/);
+    const allTiles = screen.getAllByTestId(/tile-(?!grid\b)/);
```

Comment:
There are two alternatives that might work better in this case for selecting the tiles but not the tile grid.
1. `/tile-[0-9]/` I can't remember what the tile ids are, but instead of ignoring an optional 'grid' string, just adding regex for the tile id
2. Select the tiles within the tile-grid instead of just the tile using the `within` function from RTL. `within(getByTestId('tile-grid')).getAllByTestId(/tile/i)`

Reply:
I think the second option would be the one I would choose. That query can also be added to the `ui` object. `tiles: () => within(getByTestId('tile-grid)).getAllByTestId(/tile/i)`